tests/integration/test_charm.py

```diff
@@ -84,3 +84,47 @@ async def test_connect_client_to_server(ops_test: OpsTest):
     assert (
         ret_code == 0
     ), f"Test returned code {ret_code} with stdout:\n{stdout}\nstderr:\n{stderr}"
+
+
+async def test_connect_to_console(ops_test: OpsTest):
+    """
+    Tests a deployed MinIO app by trying to connect to the MinIO console
+    """
+    # TODO: This presumes we're using microk8s.  Fix that.
+    #  could just call kubectl and count on the outside environment aliasing it for us?
+    #  or could pass kubectl path in as a variable?
```

Comment:
This [line](https://github.com/Barteus/minio-operator/blob/017629a43158abeb091fde2ee2d63b02f7384b38/.github/workflows/integrate.yaml#L51) in our github actions ensures you that you can use  just `kubectl` so you should be able to skip the microk8s part

Reply:
I will remove this TODO for both tests. This will mean I can remove the `microk8s` as a first argument from kubectl_cmd list, right?